capstone/src/main/java/com/google/univiz/scorecard/CollegeData.java

```diff
@@ -0,0 +1,33 @@
+package com.google.univiz.scorecard;
+
+import java.util.List;
+
+/**
+ * CollegeData is an interface that contains methods and functions which communicate with the
+ * CollegeScorecardAPI to obtain college data from the URL in JSON format and parse them to become
+ * ScorecardData java objects.
+ */
+public interface CollegeData {
+
+  /** TODO(biancamacias): decide what getSuggestions does */
+  public List<ScorecardData> getSuggestions(String suggestion);
```

Comment:
I think we'll want a dedicated type for autocomplete and not ScoreCard data: or- can you say more about what you intend to return here?